### ISSUE TITLE
Implement losses and metrics for multi-class segmentation tasks

### ISSUE THREAD

**Issue by josafatburmeister** [HealthML/active-segmentation#51]
- [ ] adapt model architectures to use a softmax activation layer for single-label and a sigmoid activation layer for multi-label segmentation tasks
- [ ] adapt loss functions to work with single-label and multi-label segmentation tasks
- [ ] adapt metrics to work with single-label and multi-label segmentation tasks
- [ ] exclude padding from metric and loss calculations
- [ ] implement normalised surface distance (NSD)
